kuma/static/styles/components/payments/banner.scss

```diff
@@ -213,7 +217,7 @@
     }
 
     .login-prompt {
-        padding: 20px;
+        padding: 100px 20px;
```

Comment:
This gives you a large bottom padding, too... Would `100px 20px 20px 20px` be a better choice here?

Reply:
This is desired. The design calls for the container to keep a similar height when switching from the form to the sign-in prompt, so the additional space below is to ensure it is vertically centred inside the container.